homework.md

```diff
@@ -92,19 +93,19 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send you code to Github and save it ready to be pulled into the code.
```

Comment:
Be careful. Commit your changes doesn't mean that you send your code to github. You can commit your changes without sending your code to github. If you want to send your code to github, this is done with the `git push` command after commiting your changes.
Committing your changes means:
1) adding the modified file to the staging area with a `git add` command.
2) creating a commit with that change `git commit`